pkg/kritis/review/review.go

```diff
@@ -123,7 +122,7 @@ func (r Reviewer) ReviewISP(images []string, isps []v1beta1.ImageSecurityPolicy,
 			}
 
 			glog.Infof("Getting vulnz for %s", image)
-			violations, err := r.config.Validate(isp, image, r.client)
+			violations, err := r.config.Validate(isp, image, r.client, auths)
```

Comment:
auths are in fact not needed here.  Please see comment above in securitypolicy.go.